### PR TITLE
docs: added readme

### PR DESCRIPTION
**Pull request by heueristik**
Task [OS-485](https://aragonassociation.atlassian.net/browse/OS-485)

[OS-485]: https://aragonassociation.atlassian.net/browse/OS-485?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,203 @@
...
+
+### Building
+
+First build the contracts and
+
+```sh
+yarn build
+```
+
+and generate the [typechain TypesCript bindings](https://github.com/dethcrypto/TypeChain) with
```
Comment (AaronAbuUsama) [inline README.md]:
```suggestion
and generate the [typechain TypesScript bindings](https://github.com/dethcrypto/TypeChain) with
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,203 @@
...
+
+To test your contracts, run
+
+```sh
+yarn test
+```
+
+### Linting
+
+Lint the Solidity and TypeScrip code alltogether with
```
Comment (AaronAbuUsama) [inline README.md]:
```suggestion
Lint the Solidity and TypeScript code all together with
```

File: README.md [hunk 3]
```diff
@@ -1,4 +1,203 @@
...
+
+### Deployment
+
+Deploy the contracts to the local Hardhat Network with
+
+```sh
+yarn deploy
+```
+
+To deploy to a network such as the sepolia testnet, use
```
Comment (AaronAbuUsama) [inline README.md]:
```suggestion
To deploy to a network such as the Sepolia Testnet, use
```

File: README.md [hunk 4]
```diff
@@ -1,4 +1,203 @@
...
+[hardhat-badge]: https://img.shields.io/badge/Built%20with-Hardhat-FFDB1C.svg
+[license]: https://opensource.org/licenses/AGPL-v3
+[license-badge]: https://img.shields.io/badge/License-AGPL_v3-blue.svg
+
+## Quickstart
+
+After [creating a new repository from this template](https://github.com/new?template_name=osx-plugin-template-hardhat&template_owner=aragon), cloning, and opening it in your IDE, run
+
+```sh
+yarn install && cd packages/contracts && yarn install && yarn build && yarn typechain
```
Comment (AaronAbuUsama) [inline README.md:13]: could we not have this as a script in the root json?

File: README.md (hunk 1 above)
Comment (heueristik) [reply README.md]: Fixed in 81d88ad94b115a0558ffd7895d45eb6951420467

File: README.md (hunk 2 above)
Comment (heueristik) [reply README.md]: Fixed in 81d88ad94b115a0558ffd7895d45eb6951420467

File: README.md (hunk 3 above)
Comment (heueristik) [reply README.md]: Fixed in 81d88ad94b115a0558ffd7895d45eb6951420467

File: README.md (hunk 4 above)
Comment (heueristik) [reply README.md:13]: I leave that to you or @mathewmeconry to decide. One person, either owning this repo or being responsible for our build infrastructure should make this decision and try to find the optimum between maintainability and developer friendliness.

**Review by AaronAbuUsama — APPROVED** [review]

**Comment by AaronAbuUsama** [comment]
im not authorised to merge this

**Comment by heueristik** [comment]
> im not authorised to merge this

I added you to the repo now

**Pull request merged**